src/staging/README.md

```diff
@@ -1,99 +1,12 @@
-<!-- AUTO-GENERATED-CONTENT:START (STARTER) -->
-<p align="center">
-  <a href="https://www.gatsbyjs.com">
-    <img alt="Gatsby" src="https://www.gatsbyjs.com/Gatsby-Monogram.svg" width="60" />
-  </a>
-</p>
-<h1 align="center">
-  Gatsby's default starter
-</h1>
+## Prerequisites
 
-Kick off your project with this default boilerplate. This starter ships with the main Gatsby configuration files you might need to get up and running blazing fast with the blazing fast app generator for React.
+1. Make sure Strapi is running. Follow the steps found in ``src/cms/``
+2. Copy ``.env.example`` as ``.env``.
+  * ```REACT_APP_CMS_BASE_URL=http://localhost:1337``` must be set. Be sure your Strapi server is actually running on port ``1337``.
 
-_Have another more specific idea? You may want to check out our vibrant collection of [official and community-created starters](https://www.gatsbyjs.com/docs/gatsby-starters/)._
+## Running local dev server
 
-## 🚀 Quick start
+1. ``yarn``
+2. ``yarn start``
```

Comment:
Maybe note the develop command as well for reloading?

Reply:
The start command runs the develop command

Reply:
```  "scripts": {
    "build": "gatsby build",
    "develop": "gatsby develop -H 0.0.0.0",
    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
    "start": "npm run develop",
    "serve": "gatsby serve",
    "clean": "gatsby clean",
    "test": "jest"
  },```

Start should run develop which runs gatsby develop -H 0.0.0.0

Reply:
Ah, my mistake, got it mixed up with serve. Start is fine then.

Bit odd though to call npm via npm or yarn.. 🧐